main/acle.md

```diff
@@ -4746,6 +4765,56 @@ stored to memory is modified by replacing the low 32 bits of
 `value.val[0]` with the contents of the `ACCDATA_EL1` system register.
 The returned value is the same as for `__arm_st64bv`.
 
+## Atomic store with PCDPHINT intrinsics
+
+These intrinsics provide an atomic store intrinsic, which will
+make use of the `STSHH` hint instruction immediately followed by the
+associated store instruction.
+This intrinsic is available when `__ARM_FEATURE_PCDPHINT` is defined.
+
+To access these intrinsics, `<arm_acle.h>` should be included.
+
+``` c
+  void __arm_stshh_u8(void const volatile *addr,
```

Comment:
The target of a store shouldn't be `const` qualified? And why is it `volatile` qualified?
And why not use the proper data type for the target pointer?
GCC atomic builtins use plain `int` for the memorder parameter. See https://gcc.gnu.org/onlinedocs/gcc/_005f_005fatomic-Builtins.html
`void __arm_stshh_u8(uint8_t *addr, uint8_t data, int memorder, int retpol);`